### ISSUE TITLE
How to fix typos

### ISSUE THREAD

**Issue by adriatic** [mattduffield/frontendcreator#6]
Gitbook provides some basic means to add comments:

![image](https://user-images.githubusercontent.com/2712405/27183862-2acc6f88-51ae-11e7-83c3-6da77e607d43.png)

Would you prefer your readers to use this method for reporting typos - or to report them via https://github.com/mattduffield/frontendcreator/issues ?

**Comment by mattduffield** [comment]
Not sure what is the best route but perhaps centralizing everything under the issues would make managing it easier? I am open to either as long as it doesn't get lost.

**Comment by jawa-the-hutt** [comment]
I would vote for leaving it in the issues as for something like this.

Since Gitbook is reading the markdown files in this repository, someone could just submit a pull request of the source file and once approved and committed, Gitbook would update automatically.

**Comment by adriatic** [comment]
I agree with both votes as using the issues allows screenshots where it might be easier to show the typo **and** suggest the alternatives.

**Comment by mattduffield** [comment]
Agreed. That makes sense to me as well. Thanks!
